Jenkinsfile

```diff
@@ -172,12 +172,19 @@ pipeline {
                     }
                     post { always { retry(3) { deleteDir() } } }
                 }
-                stage('Windows Headers & Unit') {
+                stage('Windows Headers') {
```

Comment:
What are this changes to the Jenkins files for?

Reply:
Looks like a bad merge with develop - we changed to "Windows Headers & Unit" on develop in the meantime

Reply:
Yeah, I messed up and also got in some gp_cov_exp stuff. My bad.